### PR TITLE
Reorg of early content

### PR DESCRIPTION
**Pull request by briandoyle81**
Change links in most edited docs to reference-style
Edit pass on `Differences Vs. EVM` to reflect Flow EVM
Remove Cadence doc that is duped on cadence-lang.org and instead link to that
Rename and reorganize some files
Add redirects to reference/fix file names changed in reorg
Change some .mdx to .md for consistency (Docusaurus compiles all .md to .mdx)
Fix links to referenced .mdx files

### CODE REVIEW

File: docs/build/smart-contracts/learn-cadence.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+sidebar_position: 1
+slug: /build/learn-cadence
+title: Learn Cadence ↗️
+---
+
+import {Redirect} from '@docusaurus/router';
```
Comment (nialexsan) [inline docs/build/smart-contracts/learn-cadence.md:7]: this can be removed
```suggestion
```

File: docs/build/differences-vs-evm/index.md [hunk 2]
```diff
@@ -172,3 +186,24 @@ If you’re already familiar with blockchain development, here's a comparison be
   - [Cadence testing framework](https://cadence-lang.org/docs/testing-framework) enables native tests in Cadence.
   - [overflow](https://github.com/bjartek/overflow) for testing in Go.
   - [js-testing](https://github.com/onflow/flow-js-testing) for testing in JS.
+
+[Why Flow]: ../flow.md
+[EVM]: ../../evm/about
+[accounts on Ethereum]: https://ethereum.org/en/developers/docs/accounts/
```
Comment (chasefleming) [inline docs/build/differences-vs-evm/index.md:192]: "accounts" needs to be capitalized

File: docs/build/differences-vs-evm/index.md [hunk 3]
```diff
@@ -172,3 +186,24 @@ If you’re already familiar with blockchain development, here's a comparison be
...
+[Flow CLI]: ../../tools/flow-cli/accounts/create-accounts.md
+[Flow account model]: ../basics/accounts.md
+[Accounts]: ../basics/accounts.md
+[storage]: http://localhost:3000/build/basics/accounts#storage
```
Comment (chasefleming) [inline docs/build/differences-vs-evm/index.md:196]: caps

File: docs/build/differences-vs-evm/index.md [hunk 4]
```diff
@@ -172,3 +186,24 @@ If you’re already familiar with blockchain development, here's a comparison be
...
+[Cadence]: https://cadence-lang.org/
+[Resources]: https://cadence-lang.org/docs/language/resources
+[Capability-based Security]: https://en.wikipedia.org/wiki/Capability-based_security
+[Entitlements]: https://cadence-lang.org/docs/1.0/language/access-control#entitlements
+[Capability-based Access Control]: https://cadence-lang.org/docs/language/capabilities
+[Guide for Solidity Developers](https://cadence-lang.org/docs/solidity-to-cadence)
+[The Cadence tutorial]: https://cadence-lang.org/docs/tutorial/first-steps
+[transaction nonce]: https://ethereum.org/en/developers/docs/accounts/#an-account-examined
```
Comment (chasefleming) [inline docs/build/differences-vs-evm/index.md:204]: caps

**Review by chasefleming — CHANGES_REQUESTED** [review]
It looks like some of the recent changes on main are missing or overwritten. For example, the recent changes from this PR to pre populate the hello world script so they don't have to copy it aren't there: https://github.com/onflow/docs/pull/962/files

Maybe try merging main into this PR

File: docs/build/differences-vs-evm/index.md (hunk 3 above)
Comment (nialexsan) [reply docs/build/differences-vs-evm/index.md:196]: fix localhost to an actual link

File: docs/build/differences-vs-evm/index.md [hunk 5]
```diff
@@ -172,3 +186,24 @@ If you’re already familiar with blockchain development, here's a comparison be
   - [Cadence testing framework](https://cadence-lang.org/docs/testing-framework) enables native tests in Cadence.
   - [overflow](https://github.com/bjartek/overflow) for testing in Go.
   - [js-testing](https://github.com/onflow/flow-js-testing) for testing in JS.
+
+[Why Flow]: ../flow.md
```
Comment (nialexsan) [inline docs/build/differences-vs-evm/index.md:190]: use md format for links 
- [Why Flow](../flow.md)

File: docs/build/differences-vs-evm/index.md [hunk 6]
```diff
@@ -172,3 +186,24 @@ If you’re already familiar with blockchain development, here's a comparison be
...
+[EVM]: ../../evm/about
```
Comment (nialexsan) [inline docs/build/differences-vs-evm/index.md:191]: add extension to the path

File: docs/build/flow.md [hunk 7]
```diff
@@ -40,20 +80,41 @@ The Flow blockchain implements core functionality using its own smart contract l
...
+
+- The [Flow Technical Primer] is a great place to start to understand how Flow works.
+- The [Three technical whitepapers] cover the unique innovation behind the Flow blockchain network in-depth.
+
+### Tokenomics
+
+- To understand more about Flow's Token Economics, and the FLOW token, read the [Flow Token Economics] guide.
+- FLOW tokens are Flow's native Fungible Token. To learn more about how to work with them in your applications, review the [FLOW] article.
+
+[digital assets]: https://www.onflow.org/post/flow-blockchain-cadence-programming-language-resources-assets
```
Comment (nialexsan) [inline docs/build/flow.md:97]: use md format for links

File: docs/build/getting-started/fcl-quickstart.md [hunk 8]
```diff
@@ -157,10 +157,10 @@ Run `npm start` again.  After a moment, the greeting from `HelloWorld` will appe
 
 For a deeper dive into writing an FCL app, such as how to change the chain state with FCL, check out [the app quickstart guide] or the [FCL documentation].
 
-[Flow Client Library]: ../../../tools/clients/fcl-js
+[Flow Client Library]: ../../tools/clients/fcl-js
```
Comment (nialexsan) [inline docs/build/getting-started/fcl-quickstart.md:160]: use md format for links

**Review by nialexsan — CHANGES_REQUESTED** [review]
fix links addresses and links format

File: docs/build/smart-contracts/learn-cadence.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
+---
+sidebar_position: 1
+slug: /build/learn-cadence
+title: Learn Cadence ↗️
```
Comment (nialexsan) [inline docs/build/smart-contracts/learn-cadence.md:4]: add `redirect` front matters

Comment (briandoyle81) [reply docs/build/smart-contracts/learn-cadence.md:4]: @nialexsan that didn't work.  The router treated it as a relative link and was constructing the link as `https://docs.cadence.org/https://cadence-lang.org/docs/`

Is there a way to tell the router in the frontmatter that it's an outbound link?

**Comment by briandoyle81** [comment]
Resolves #953

File: docs/build/differences-vs-evm/index.md (hunk 2 above)
Comment (briandoyle81) [reply docs/build/differences-vs-evm/index.md:192]: Relative-style links, added comment to clarify!

File: docs/build/differences-vs-evm/index.md [hunk 10]
```diff
@@ -1,48 +1,60 @@
...
 
 ![Screenshot 2023-08-16 at 16.43.07.png](../basics/_accounts_images/Screenshot_2023-08-16_at_16.43.07.png)
 
 This natively enables interesting use cases, like key revocation, rotation, and multi-signature transactions. All Flow accounts can use network storage (e.g., for deploying contracts and storing resources like NFTs) based on the number of FLOW tokens they hold.
 
-<Callout type="info">
+:::warning
 
-You must run an explicit account creation transaction on Flow to create a new account. [Flow CLI](../../tools/flow-cli/accounts/create-accounts.md) can create an account on any network with a given public key.
+You must run an explicit account creation transaction on Flow to create a new account. [Flow CLI] can create an account on any network with a given public key.
```
Comment (gregsantos) [inline docs/build/differences-vs-evm/index.md:30]: might want to add a note that a small fee is required to be paid in FLOW for creation

File: docs/build/differences-vs-evm/index.md [hunk 11]
```diff
@@ -53,9 +65,9 @@ If you are a Solidity developer, it's recommended to read Cadence's [Guide for S
   - [Tutorial](https://cadence-lang.org/docs/tutorial/marketplace-setup)
   - [NFT Storefront](https://github.com/onflow/nft-storefront/) is an example marketplace standard
 
-### Transactions and Scripts
+## Transactions and Scripts
 
-You can interact with the state on most other blockchains by cryptographically authorizing smart contract function calls. On Flow, transactions are more complex and are pieces of Cadence code. This means that any number of contracts and function calls can be composed together atomically to mutate the blockchain state.
+You can interact with the state on most other blockchains by cryptographically authorizing smart contract function calls. On Flow, transactions are more complex and are written with Cadence code. This means that any number of contracts and function calls can be composed together atomically to mutate the blockchain state.
```
Comment (gregsantos) [inline docs/build/differences-vs-evm/index.md:70]: can we say this a bit differently? Maybe something like:
"On Flow, transactions offer rich functionality through Cadence code. This allows you to seamlessly combine multiple contracts and function calls into a single transaction that updates the blockchain state - all executing together as one unified operation."

File: docs/build/flow.md
Comment (nialexsan) [inline docs/build/flow.md:1]: revert links changes to include extensions

**Review by nialexsan — APPROVED** [review]
non-blocking comment

**Review by chasefleming — APPROVED** [review]

**Pull request merged**